lgc/include/lgc/patch/FragColorExport.h

```diff
@@ -64,6 +64,9 @@ class FragColorExport {
                                   llvm::ArrayRef<ExportFormat> exportFormat, bool dummyExport, BuilderBase &builder);
   static void setDoneFlag(llvm::Value *exportInst, BuilderBase &builder);
   static llvm::CallInst *addDummyExport(BuilderBase &builder);
+  static llvm::Function *generateNullFragmentShader(llvm::Module &module, llvm::StringRef entryPointName);
+  static llvm::Function *generateNullFragmentEntryPoint(llvm::Module &module, llvm::StringRef entryPointName);
+  static void generateNullFragmentShaderBody(llvm::Function *entryPoint);
```

Comment:
I have moved these function to the FragColorExport class because they are related to the color exports.  They cannot be accessed from `PatchNullFragShader` because its header file is not in the include directory.  I do not have strong feelings about where they should live, let me know if you want them somewhere else.

Reply:
Seems fine where they are.